egs2/opencpop/svs1/conf/tuning/train_singing_tacotron.yaml

```diff
@@ -0,0 +1,86 @@
+    aconv_filts: 15              # filter size of convolutional layer of attention
+    cumulate_att_w: true         # whether to cumulate attention weight
+    dlayers: 2                   # number of lstm layers in decoder
+    dunits: 1024                 # number of lstm units in decoder
+    prenet_layers: 2             # number of layers in prenet
+    prenet_units: 256            # number of units in prenet
+    postnet_layers: 5            # number of layers in postnet
+    postnet_chans: 512           # number of channels in postnet
+    postnet_filts: 5             # filter size of postnet layer
+    use_batch_norm: true         # whether to use batch normalization in encoder
+    use_concate: true            # whether to concatenate encoder embedding with decoder outputs
+    use_residual: false          # whether to use residual connection in encoder
+    reduction_factor: 1          # reduction factor
+    use_gst: False
+    gst_tokens: 10
+    gst_heads: 4
+    gst_conv_layers: 6
+    gst_conv_kernel_size: 3
+    gst_conv_stride: 2
+    gst_gru_layers: 1
+    gst_gru_units: 128
+    dropout_rate: 0.5
+    zoneout_rate: 0.1
+    use_masking: True
+    use_weighted_masking: False
+    bce_pos_weight: 5.0
+    loss_type: L1
+    use_guided_attn_loss: true   # whether to use guided attention loss
+    guided_attn_loss_sigma: 0.4  # sigma of guided attention loss
+    guided_attn_loss_lambda: 1.0 # strength of guided attention loss
+
+
+##########################################################
+#                  OPTIMIZER SETTING                     #
+##########################################################
+optim: adam           # optimizer type
+optim_conf:           # keyword arguments for selected optimizer
+    lr: 1.0e-03       # learning rate
+    eps: 1.0e-06      # epsilon
+    weight_decay: 0.0 # weight decay coefficient
+
+##########################################################
+#                OTHER TRAINING SETTING                  #
+##########################################################
+# num_iters_per_epoch: 200    # number of iterations per epoch
+max_epoch: 500              # number of epochs
+grad_clip: 1.0              # gradient clipping norm
+grad_noise: false           # whether to use gradient noise injection
+accum_grad: 1               # gradient accumulation
+
+batch_size: 16
+batch_type: sorted          # how to make batch
+
+sort_in_batch: descending   # how to sort data in making batch
+sort_batch: descending      # how to sort created batches
+num_workers: 10             # number of workers of data loader
+train_dtype: float32        # dtype in training
+log_interval: 1             # log interval in iterations
+keep_nbest_models: 2        # number of models to keep
+num_att_plot: 3             # number of attention figures to be saved in every check
+seed: 0                     # random seed number
+best_model_criterion:
+-   - valid
+    - loss
```

Comment:
Have you tried instead use total_counts as the criterion?

Reply:
I haven't tried this.

Reply:
Yeah, for TTS model, it might be good to use total_count (maybe we can have a check if that would be better) as loss is not a very good metric